e2e-tests/cypress/tests/integration/channels/system_console/user_management_spec.js

```diff
@@ -222,6 +222,17 @@ describe('User Management', () => {
         cy.url().should('contain', '/login');
     });
 
+    it("MM-58840 Users - can't navigate to invalid URL", () => {
+        // # Login as sysadmin.
+        cy.apiLogin(sysadmin);
+
+        // # Visit the invalid URL.
+        cy.visit('/admin_console/user_management/user/invalid');
+
+        // * Verify that the user is redirected to the default page.
+        cy.url().should('include', '/admin_console/about/license');
```

Comment:
this one is handled differently compared to the other ones use routes like `/groups` for listing  and `/groups/{id}` for detail, but user uses `/users` for listing but `/user/{id}` (no **S** after user) to show a user.